ncm-download/pom.xml

```diff
@@ -1,16 +1,16 @@
-<?xml version="1.0"?>
+<?xml version="1.0" encoding="utf-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.quattor.cfg.module</groupId>
   <artifactId>download</artifactId>
   <packaging>pom</packaging>
   <version>17.3.1-SNAPSHOT</version>
-  <name>Morgan Stanley ncm-download component</name>
+  <name>ncm-download: Morgan Stanley ncm-download componen</name>
```

Comment:
what's so MS specific about this? 
@ned21 @gombasg do yo umind if we remnove MS?